src/test/ui/closures/2229_closure_analysis/diagnostics/liveness.rs

```diff
@@ -0,0 +1,63 @@
+// edition:2018
```

Comment:
1) remove edition2018 from this
2) liveness executable got checked in, please remove it